packages/eslint-plugin/src/rules/naming-convention.ts

```diff
@@ -538,6 +540,16 @@ export default createRule<Options, MessageIds>({
           },
         },
 
+      AccessorProperty: {
+        validator: validators.autoAccessor,
+        handler: (
+          node: TSESTree.AccessorPropertyNonComputedName,
+          validator,
+        ): void => {
+          const modifiers = getMemberModifiers(node);
+          handleMember(validator, node, modifiers);
+        },
+      },
       // #endregion accessor
```

Comment:
[Style] Nit: `#region accessor` refers to the existing accessor stuff. This auto-accessor / `AccessorProperty` should be in a new region IMO.

Reply:
Ye, added.